### PR TITLE
Add Total Staking to Homa Storage

### PR DESCRIPTION
**Pull request by ferrell-code**
closes: #2112 closes: #2084

### CODE REVIEW

**Review by ermalkaleci — DISMISSED** [review]
LGTM

File: modules/homa/src/lib.rs [hunk 1]
```diff
@@ -627,10 +635,20 @@ pub mod module {
...
 					*maybe_ledger = if ledger == Default::default() {
+						TotalStakingBonded::<T>::mutate(|staking_balance| {
+							*staking_balance = staking_balance.saturating_sub(bonded_amount)
+						});
 						None
 					} else {
+						TotalStakingBonded::<T>::mutate(|staking_balance| {
+							*staking_balance = staking_balance
+								.saturating_add(ledger.bonded)
+								.saturating_sub(bonded_amount)
```
Comment (zjb0807) [inline modules/homa/src/lib.rs:650]:
```suggestion
```

Comment (zjb0807) [reply modules/homa/src/lib.rs:650]: The code should be deleted here. What did I miss?

Comment (ferrell-code) [reply modules/homa/src/lib.rs:650]: It is `TotalStakingBonded` + updated ledger bonded - old ledger bonded.  

Naming it `old_bonded_amount` may be more clear

**Review by zjb0807 — APPROVED** [review]

**Review by wangjj9219 — APPROVED** [review]

**Pull request merged**